### DISCUSSION TITLE
Not Seeing inswapper on fooocus UI

### DISCUSSION THREAD

**Discussion by Icarushollow** [lllyasviel/Fooocus#3530] [Q&A]
Hi, I did the initial steps of installing the 2 C++ requirements then executed the the .git link on my cmd and I'm not seeing the inswapper checkbox or anything of the sort. I restarted my PC afterwards to apply the install as suggested. One thing that could be the issue is the inswapper folder ended up in my c://user... files. Maybe it needed to be installed somewhere else? Not exactly sure but would love to give this a try, I'm just not able to get it on my fooocus UI. Any help would be appreciated

**Comment by IPv6** [comment]
Fooocus does not support it, so no checkboxes

You need a fork for it, like for example that one - https://github.com/machineminded/Fooocus-inswapper
